### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by indytechcook**
@tladendo and @vietnogi just a few small fixes required for the build to be happy

### CODE REVIEW

**Comment by tladendo** [comment]
lgtm

File: gulpfile.js [hunk 1]
```diff
@@ -76,11 +76,14 @@ gulp.task('scss', function () {
 gulp.task('jade', function () {
   log('Compiling Jade --> HTML');
 
-  var stubs = config.useStubs ? true : false;
+  var stubs = config.useStubs === 'true';
```
Comment (indytechcook) [inline gulpfile.js]: this needs to be literal because of how env variable are read where 'false' is pulled in as a string

**Comment by indytechcook** [comment]
@tladendo and @vietnogi this is ready to go.  It includes the image path fixes

File: gulpfile.js (hunk 1 above)
Comment (vietnogi) [inline gulpfile.js]: Is there a way to handle stubs without intruding with jade files?  Don't want to intertwine the two layers and have to decouple them later.

Comment (indytechcook) [inline gulpfile.js]: We previously had them being injected but changed to this method because we discussed wanted to be as transparent as possible.

Comment (vietnogi) [inline gulpfile.js]: Is there an alternative solution for stubbing that is abstracted out of our development code.

Comment (indytechcook) [inline gulpfile.js]: I'm not sure what you are asking.  Stubs are used in local development.  They are either in the jade or they are injected.

Comment (vietnogi) [inline gulpfile.js]: Currently, you're coupling a stub solution to our source code which is poor design, it should be abstracted out in its own layer.  For example Compass vs Bourbon/Bitters/Refills.

**Comment by indytechcook** [comment]
Merging, @tladendo is working stub stuff in another request

**Pull request merged**